src/Bicep.Core.Samples/Files/baselines/Modules_CRLF/main.json

```diff
@@ -1,2054 +1,2054 @@
-              "templateHash": "1347091426241151379"
-            }
-          },
-          "parameters": {
-            "location": {
-              "type": "string",
-              "defaultValue": "westus"
-            }
-          },
-          "resources": [],
-          "outputs": {
-            "loc": {
-              "type": "string",
-              "value": "[parameters('location')]"
-            }
-          }
-        }
-      }
-    },
-    {
-      "type": "Microsoft.Resources/deployments",
-      "apiVersion": "2022-09-01",
-      "name": "withSeparateConfig",
-      "properties": {
-        "expressionEvaluationOptions": {
-          "scope": "inner"
-        },
-        "mode": "Incremental",
-        "template": {
-          "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
-          "languageVersion": "2.1-experimental",
-          "contentVersion": "1.0.0.0",
-          "metadata": {
-            "_EXPERIMENTAL_WARNING": "This template uses ARM features that are experimental. Experimental features should be enabled for testing purposes only, as there are no guarantees about the quality or stability of these features. Do not enable these settings for any production usage, or your production environment may be subject to breaking.",
-            "_EXPERIMENTAL_FEATURES_ENABLED": [
-              "Extensibility"
-            ],
-            "_generator": {
-              "name": "bicep",
-              "version": "dev",
-              "templateHash": "8139539847401806108"
-            }
-          },
-          "imports": {
-            "az": {
-              "provider": "AzureResourceManager",
-              "version": "1.0.0"
-            }
-          },
-          "resources": {},
-          "outputs": {
-            "str": {
-              "type": "string",
-              "value": "foo"
-            }
-          }
-        }
-      }
-    }
-  ],
-  "outputs": {
-    "stringOutputA": {
-      "type": "string",
-      "value": "[reference(resourceId('Microsoft.Resources/deployments', 'modATest'), '2022-09-01').outputs.stringOutputA.value]"
-    },
-    "stringOutputB": {
-      "type": "string",
-      "value": "[reference(resourceId('Microsoft.Resources/deployments', 'modATest'), '2022-09-01').outputs.stringOutputB.value]"
-    },
-    "objOutput": {
-      "type": "object",
-      "value": "[reference(resourceId('Microsoft.Resources/deployments', 'modATest'), '2022-09-01').outputs.objOutput.value]"
-    },
-    "arrayOutput": {
-      "type": "array",
-      "value": "[reference(resourceId('Microsoft.Resources/deployments', 'modATest'), '2022-09-01').outputs.arrayOutput.value]"
-    },
-    "modCalculatedNameOutput": {
-      "type": "object",
-      "value": "[reference(resourceId('Microsoft.Resources/deployments', format('{0}{1}', 'optionalWithAllParamsAndManualDependency', parameters('deployTimeSuffix'))), '2022-09-01').outputs.outputObj.value]"
-    }
-  }
+{

```

Comment:
What's causing this diff?

Reply:
https://github.com/Azure/bicep/pull/12267/files#diff-cabd4df0f2a32b0337bf45726cdd53d9cdf7c540ef86d76438de98c7367f0a71R193